fabric-crash-report-info-v1/src/main/resources/fabric.mod.json

```diff
@@ -1,23 +1,20 @@
 {
   "schemaVersion": 1,
   "id": "fabric-crash-report-info-v1",
-  "name": "Fabric Crash Report Info (v1)",
+  "name": "Legacy Fabric Crash Report Info (v1)",
   "version": "${version}",
   "environment": "*",
   "license": "Apache-2.0",
   "icon": "assets/fabric-crash-report-info-v1/icon.png",
   "contact": {
-    "homepage": "https://fabricmc.net",
-    "irc": "irc://irc.esper.net:6667/fabric",
-    "issues": "https://github.com/FabricMC/fabric/issues",
-    "sources": "https://github.com/FabricMC/fabric"
+    "homepage": "https://legacyfabric.net",
+    "irc": "irc://irc.esper.net:6667/legacyfabric",
+    "issues": "https://github.com/Legacy-Fabric/fabric/issues",
+    "sources": "https://github.com/Legacy-Fabric/fabric"
   },
   "authors": [
-    "FabricMC"
+    "Legacy Fabric"
   ],
-  "depends": {
-    "fabricloader": ">=0.4.0"
```

Comment:
why?

Reply:
because `patch-1` is not semver and it breaks comparison